mesa/visualization/solara_viz.py

```diff
@@ -173,24 +178,17 @@ def ComponentsView(
 
 
 @solara.component
-def ModelController(model: solara.Reactive[Model], play_interval=100):
+def ModelController(model: solara.Reactive[Model], model_parameters, play_interval=100):
```

Comment:
Can you insert a `*` here to make model_parameters and play_interval keyword only? Less arguing about the order of arguments then. Also: can we default this to None and then create an empty (reactive) dict if not provided?